lib/merchant.rb

```diff
@@ -0,0 +1,15 @@
+class Merchant
+  # This is the Merchant Class
+  # frozen_string_literal: true.
+  attr_reader :id,
+              :name
+
+  def initialize(data)
+    @id   = data[:id]
+    @name = data[:name]
```

Comment:
Brandon and I are immediately taking the data and creating merchants/items/etc in our Repos initialize methods. We can revisit this, both work.